### PR TITLE
support more actions, feedback, and presets

### PR DESCRIPTION
**Pull request by ddabela**
I have added the additional actions, feedbacks, and presets to support the next version of "Auto Presentation Switcher".

### CODE REVIEW

File: feedbacks.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+var choices = require('./choices');
+exports.getFeedbacks = function (instance) {
+	var self = instance;
+	return {
+		feedback_loaded: {
```
Comment (krocheck) [inline feedbacks.js:5]: In general I'd like to see these feedback keys have the `feedback_` portion removed since its redundant

File: index.js [hunk 2]
```diff
@@ -63,9 +73,53 @@ instance.prototype.initTCP = function () {
...
+            // data is Buffer object
+            try {
+                // console.log(data.toString('utf8'));
+                let jsonData = JSON.parse(data.toString('utf8'));
+                // console.log(jsonData);
+
+                if (jsonData.action === "states") {
+                    states.updateStates(self.displayStates, jsonData.data);
+                    // console.log(self.displayStates);
+                    self.checkFeedbacks("feedback_loaded");
```
Comment (krocheck) [inline index.js:92]: These can be combined into a single call now: `self.checkFeedbacks('loaded', 'displayed')` (note I removed the feedback_ like earlier recommended.

File: package.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "presentationtools-aps",
-  "version": "1.0.0",
-  "api_version": "1.0.0",
+  "version": "2.0.0",
+  "api_version": "2.0.0",
```
Comment (krocheck) [inline package.json:4]: `api_version` is a companion thing and needs to remain at 1.0.0.  for `version`, this feels more like a 1.1.0 to me.  Thoughts?

File: presets.js [hunk 4]
```diff
@@ -0,0 +1,941 @@
+exports.getPresets = function (instance) {
```
Comment (krocheck) [inline presets.js:1]: Any of these you can accomplish using loops?

File: feedbacks.js (hunk 1 above)
Comment (ddabela) [reply feedbacks.js:5]: Ok, I will remove the "feedback_" portion in the key.

File: index.js (hunk 2 above)
Comment (ddabela) [reply index.js:92]: It's great if we can make a single call. I will change it as your guide.

File: package.json (hunk 3 above)
Comment (ddabela) [reply package.json:4]: 1.1.0 for the version would be better. I will change.

File: presets.js (hunk 4 above)
Comment (ddabela) [reply presets.js:1]: I will reduce many lines using loops.

**Comment by ddabela** [comment]
Ok, just pushed changes after cleaned code as you indicated above.

**Pull request merged**